corehq/apps/users/migrations/0032_add_download_reports_permission.py

```diff
@@ -1,18 +1,18 @@
+import traceback
 from django.db import migrations
 
-from corehq.apps.users.models_sql import SQLPermission, SQLUserRole
+from corehq.apps.couch_sql_migration.tests.test_migration import call_command
 from corehq.util.django_migrations import skip_on_fresh_install
 
 
 @skip_on_fresh_install
 def migrate_download_reports_permissions(apps, schema_editor):
-    new_permission_name = 'download_reports'
-    permission, created = SQLPermission.objects.get_or_create(value=new_permission_name)
-    for role in SQLUserRole.objects.all().iterator():
-        rp, created = role.rolepermission_set.get_or_create(permission_fk=permission,
-                                                            defaults={"allow_all": True})
-        if created:
-            role._migration_do_sync()
+    try:
+        call_command(
```

Comment:
not blocking but you could use [run_once_off_migration](https://github.com/dimagi/commcare-hq/blob/ff287722472550c1277f19f043ca5bd6b4bd4027/corehq/util/django_migrations.py#L118)